app/workflows/validation/check_legislation_description_task.rb

```diff
@@ -21,7 +17,8 @@ def task_list_status
     end
 
     def render?
-      planning_application.application_type.legislation_description
+      application_type = planning_application.application_type
+      application_type.legislative_requirements? && application_type.legislation_description
```

Comment:
Should this have a `.present?`? Not sure if it's really likely ever to be non-nil but empty, admittedly.